Libraries/xcsdk/Tests/test_PlatformVersion.cpp

```diff
@@ -37,8 +37,8 @@ TEST(PlatformVersion, Parse)
     auto platformVersion = PlatformVersion::Open(&filesystem, "/" + platform);
```

Comment:
For all the new properties you've added parsing support for, can you write a quest test to make sure they're represented?